packages/node_modules/@ciscospark/react-component-activity-item-base/src/styles.css

```diff
@@ -44,10 +52,23 @@
   flex: 0 0 auto;
 }
 
-.pending {
```

Comment:
Had to change the pending opacity to each element because we didn't want the error message to opaque even though it was a sub element of `.pending`